### PR TITLE
Feature/updates

### PR DESCRIPTION
**Pull request by iruzevic**
- updating components

### CODE REVIEW

File: blocks/init/src/Blocks/components/button/button.php [hunk 1]
```diff
@@ -3,11 +3,12 @@
 /**
  * Template for the Button Component.
  *
- * @package EightshiftBoilerplate
+ * @package Project
  */
 
-use EightshiftBoilerplateVendor\EightshiftLibs\Helpers\Components;
+use ProjectVendor\EightshiftLibs\Helpers\Components;
```
Comment (dingo-d) [inline blocks/init/src/Blocks/components/button/button.php:9]: Same question with this one. I thought this gets changed with WPCLI scripts.

File: blocks/init/src/Blocks/components/button/button.php [hunk 2]
```diff
@@ -42,53 +43,45 @@
...
-	Components::selector($buttonAlign, "{$componentClass}-wrap", 'align', $buttonAlign),
-	Components::selector($blockClass, $blockClass, "{$selectorClass}-wrap"),
-]);
-
 $buttonClass = Components::classnames([
 	$componentClass,
-	Components::selector($buttonColor, $componentClass, 'color', $buttonColor),
-	Components::selector($buttonSize, $componentClass, 'size', $buttonSize),
-	Components::selector($buttonWidth, $componentClass, 'size-width', $buttonWidth),
+	Components::selector($buttonAsLink, $componentClass, 'as-link'),
```
Comment (dingo-d) [inline blocks/init/src/Blocks/components/button/button.php:48]: Would `is-link` be a better action modifier?

File: blocks/init/src/Blocks/components/button/button.php (hunk 1 above)
Comment (iruzevic) [reply blocks/init/src/Blocks/components/button/button.php]: she same as up

File: blocks/init/src/Blocks/components/button/button.php (hunk 2 above)
Comment (iruzevic) [reply blocks/init/src/Blocks/components/button/button.php]: updated

File: blocks/init/src/Blocks/components/paragraph/paragraph-style.scss [hunk 3]
```diff
@@ -27,14 +26,14 @@ $paragraph: (
...
 
-	a {
-		@include link(map-get-strict($paragraph, link-color), primary);
-		transition: {
-			property: color;
-			timing-function: ease-in;
-			duration: 0.3s;
-		}
-	}
+	// a {
```
Comment (dingo-d) [inline blocks/init/src/Blocks/components/paragraph/paragraph-style.scss:29]: Commented code

**Review by IvanGrginovInf — DISMISSED** [review]

**Comment by iruzevic** [comment]
I have updated paragraph component

**Review by dingo-d — APPROVED** [review]

**Pull request merged**